website/themes/uppy/layout/example.ejs

```diff
@@ -52,7 +52,7 @@
     }
 
     console.log = customLog(console.log.bind(console), document.getElementById("console-log"));
-    console.debug = customLog(console.log.bind(console), document.getElementById("console-log"));
+    console.debug = customLog(console.debug.bind(console), document.getElementById("console-log"));
```

Comment:
I think this used console.log for compatibility with IE

Reply:
@goto-bus-stop,  I thought so too (it was mentioned in comments that it's for IE10), - but that's a demo site for the Js library, I don't think anyone is developing on IE10.

Reply:
yea, just allows us to test things more easily by going to uppy.io with browserstack :)

Reply:
Could we try to switch to `log` if `debug` is unavailable like in https://github.com/transloadit/uppy/blob/master/packages/%40uppy/core/src/loggers.js? Since we override these methods here, we will not be seeing logs in the console either with current approach?

Alternatively, we could switch to the custom logging function that we now support: https://uppy.io/docs/uppy/#logger.

Reply:
@arturi, I thought of switching to the custom logger, but then we'd need to import it in every `app.es6` of every example.
Added the `log || debug` for IE10, I'll push it when my github issues resolve.

Reply:
💖